runtime/src/main/java/org/corfudb/runtime/view/LayoutBuilder.java

```diff
@@ -361,10 +347,8 @@ public LayoutBuilder removeLogunitServer(String endpoint)
      *
      * @param endpoints Log unit servers to be removed
      * @return Workflow manager
-     * @throws LayoutModificationException Cannot remove non-replicated logunit
      */
-    public LayoutBuilder removeLogunitServers(Set<String> endpoints)
-            throws LayoutModificationException {
+    public LayoutBuilder removeLogunitServers(Set<String> endpoints) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 16 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)

Reply:
Done.